src/java/picard/sam/revertsam/ValidationUtil.java

```diff
@@ -0,0 +1,124 @@
+package picard.sam.revertsam;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.util.IOUtil;
+import picard.PicardException;
+import picard.util.TabbedTextFileWithHeaderParser;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.List;
+
+/**
+ * Helper functions for validating RevertSam parameters. 
+ */
+public class ValidationUtil {
```

Comment:
inner class in `RevertSam`?  Just have the methods in `RevertSam`?  package protected class?